docs/source/overview/cli.md

```diff
@@ -0,0 +1,499 @@
+# Selene CLI operations and outputs
+Selene provides a command-line interface (CLI) that takes in a user-specified configuration file containing the operations the user wants to run and the parameters required for these operations. (See [Operations](#Operations) for more detail.)
+
+The sections that follow describe in detail how the various components that make up the configuration file are specified. For operation-specific sections (e.g. training, evaluation), we also explain what the expected outputs are.
+
+## Table of Contents
+
+We strongly recommend you read through the first 4 sections ([Overview](#Overview), [Operations](#Operations), [General configurations](#General-configurations), and [Model architecture](#Model-architecture)) and then pick other sections based on your use case. 
+
+- [Overview](#Overview)
+- [Operations](#Operations)
+- [General configurations](#General-configurations)
+- [Model architecture](#Model-architecture) 
+- [Train](#Train)
+    - [Expected outputs for training](#Expected-outputs-for-training)
+- [Evaluate](#Evaluate)
+    - [Expected outputs for evaluation](#Expected-outputs-for-evaluation)
+- [Analyze sequences](#Analyze-sequences)
+    - [Prediction on sequences](#Predict-on-sequences)
+    - [Variant effect prediction](#Variant-effect-prediction)
+    - [_In silico_ mutagenesis](#In-silico-mutagenesis)
+- [Sampler configurations](#Sampler-configurations)
+    - Samplers for training (and evaluation, optional):
+        - [Random positions sampler](#Random-positions-sampler)
+        - [Intervals sampler](#Intervals-sampler)
+        - [Multiple-file sampler](#Multiple-file-sampler)
+    - File samplers for evaluation:
+        - [BED file sampler](#BED-file-sampler)
+        - [Matrix file sampler](#Matrix-file-sampler)
+
+## Overview
+Selene's CLI accepts configuration files that are composed of 4 main (high-level) groups:
```

Comment:
Will be nice to have some more info about the config files format, for example, "CLI accepts configuration files in [YAML] (link to outside(?) page about it) format".